docs/accounts/account-create.md

```diff
@@ -0,0 +1,45 @@
+[](){#ref-account-create}
+# Creating an account
+
+!!! warning
+    The process for creating an account documented here applies only to users who are invited using the new [project and resource management tool][ref-account-waldur] (Waldur).
+
+When the CSCS Account Manager, project PI or Deputy PI invites the users they will receive an invitation email if the invited user has an **existing** CSCS account then the user clicks on the URL from the email and log-in with a username, password, OTP, and accept the invitation **whereas** if the invited user is new then the user needs to follow the step-by-step instructions below to get an account
+
+The email contains a url that redirects you to the registratin page:
+
+![](../images/access/invitation.png)
+
+Clicking the "Create a new account" button will lead the user to the 2nd step where he needs to provide his personal information as shown below:
```

Comment:
```suggestion
Clicking the "Create a new account" button will lead the user to the second step where he needs to provide his personal information as shown below:
```

Reply:
Possibly:
```suggestion
Clicking the "Create a new account" button will lead the user to the 2nd step where they need to provide their personal information as shown below:
```
but that's opening a bigger box of fun. I have no idea what we use elsewhere...

Edit: seems like the rest of the document talks to "you" as the user. I'm not blocking on this, but if you feel like making it consistent, go for it!